src/annotator/util/shadow-root.ts

```diff
@@ -36,25 +36,5 @@ export function createShadowRoot(container: HTMLElement): ShadowRoot {
     applyFocusVisible(shadowRoot);
   }
 
-  stopEventPropagation(shadowRoot);
   return shadowRoot;
 }
-
-/**
- * Stop bubbling up of several events.
- *
- * This makes the host page a little bit less aware of the annotator activity.
- * It is still possible for the host page to manipulate the events on the capturing
- * face.
```

Comment:
This comment is hand-wavy and doesn't point to any specific issues with web pages that we need to be aware of, so I'm going to ignore it. I will look back through the commit history tomorrow in case I can find any remaining reasons to keep the logic, but otherwise let's try to simplify the code.